src/modules/models/Tree/Tree.tsx

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Are we supposed to be able to generate these ?

Reply:
Yes, I added a section in the README